hy/_compat.py

```diff
@@ -14,3 +15,33 @@ def reraise(exc_type, value, traceback=None):
         raise value.with_traceback(traceback)
     finally:
         traceback = None
+
+
+if PY3_9:
+
+    import ast, math, cmath
```

Comment:
The `math` and `cmath` imports appear irrelevant.

Reply:
Yep, they're leftovers.